mne/decoding/search_light.py

```diff
@@ -0,0 +1,491 @@
+# Author: Jean-Remi King <jeanremi.king@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+
+from .mixin import TransformerMixin
+from .base import BaseEstimator  # XXX why are these sklearn object copied in weird places. We should just have an externals.sklearn? # noqa
```

Comment:
@agramfort @dengemann Do you have a clear idea these imports? They've been copied/pasted without being explicitly put in an external directory, which will make the sklearn update quite painful, no?